app/orm-adapters/node-orm2.js

```diff
@@ -7,11 +7,11 @@ import snakeCase from 'lodash/snakeCase';
 import upperFirst from 'lodash/upperFirst';
 
 const typeMap = {
-  text: String,
-  number: Number,
-  boolean: Boolean,
-  json: Object,
-  date: Date
+  text: 'text',
+  number: 'number',
+  boolean: 'boolean',
+  json: 'object',
+  date: 'date'
```

Comment:
Left side is the ORM name to expose to our Denali model.  Right side is the corresponding ORM name to use when talking to our actual ORM.  Had to change away from actual types due to the use of `mapsTo` below.